modules/schema/src/main/scala/lucuma/odb/data/BandedTime.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.data
+
+import cats.Eq
+import lucuma.core.enums.ScienceBand
+import lucuma.core.model.sequence.CategorizedTime
+
+case class BandedTime(
+  band: Option[ScienceBand],
+  time: CategorizedTime
+)
+
+object BandedTime:
+  val Empty: BandedTime = BandedTime(None, CategorizedTime.Zero)
+
+  given Eq[BandedTime] =
```

Comment:
Might be useful to define `Order` here. Looks like it's defined for the member types.

Reply:
:+1: Thanks, that's a good point.  I've updated it but will add property tests when I move this to core.